src/components/ui/CoralButton.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component,  PropTypes } from 'react';
+import Radium from 'radium';
+import color from 'color';
+
+const buildCustomStyle = (customColor) => ({
+  backgroundColor: customColor,
+  color: '#FFFFFF',
+  ':hover': {
+    backgroundColor: color(customColor).lighten(0.2).hexString()
+  },
+  active: {
+    backgroundColor: '#FFFFFF',
+    color: customColor,
+    ':hover': {
+      backgroundColor: '#FFFFFF'
+    }
+  }
+})
+
+@Radium
+class Button extends Component {
+  render() {
+    const { type = 'default', className, onClick, disabled, children, icon, style, active = false, customColor, ...rest } = this.props;
+
+    if (type === 'custom' && customColor) {
+      styles.custom = buildCustomStyle(customColor)
+    }
+
+    const finalStyles = {
+      ...styles.base,
+      ...styles[type],
+      ...style
+    };
+
+    return (
+    <button
+      className={`mdl-button mdl-js-button mdl-button--raised ${className}`}
+      onClick={onClick}
+      style={[
+        finalStyles,
+        { ':hover': { backgroundColor: color(finalStyles.backgroundColor).lighten(0.2).hexString() }},
+        active ? styles[type].active : {}
+      ]}
+      disabled={disabled ? 'disabled' : ''}
+      { ...rest }
+    >
+      { icon ? <i className="material-icons" style={styles.icon} > {icon} </i> : null }
+      { children }
+    </button>
+    )
+  }
+}
+
+const boxShadowDefault = '0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12)';
+
+let styles = {
+  base: {
+    color: 'white',
+    fontSize: '0.9em',
+    textTransform: 'none',
+    backgroundColor: 'white',
+    ':hover': {
+      backgroundColor: '#d8d8d8'
+    },
+    ':focus': {
+      boxShadow: 'none',
+      boxShadow: boxShadowDefault
+    }
+  },
+  success: {
+    backgroundColor: '#00796B'
```

Comment:
there is a file `/src/settings.js` that has all these base colors defined in it so the whole app will update (in theory) if you change constants like this. check line 42.